spec/factories/users.rb

```diff
@@ -1,7 +1,11 @@
-# This will guess the User class
 FactoryBot.define do
   factory :user do
-    email {Faker::Internet.email}
-    password {Faker::Internet.password}
+    email { Faker::Internet.email }
+    password { Faker::Internet.password }
+    confirmed_at Time.now
```

Comment:
`Time.zone.now` sets true timezone. I don't know why we need it in tests but let's develop a habit and use `Time` with `zone` everywhere.

Reply:
done